src/main/java/no/uio/ifi/localega/doa/services/StreamingService.java

```diff
@@ -97,7 +97,7 @@ private InputStream getPlaintextResponse(byte[] header, InputStream bodyInputStr
         ByteArrayInputStream headerInputStream = new ByteArrayInputStream(header);
         SequenceInputStream sequenceInputStream = new SequenceInputStream(headerInputStream, bodyInputStream);
         Crypt4GHInputStream crypt4GHInputStream;
-        if (!StringUtils.isEmpty(startCoordinate) && !StringUtils.isEmpty(endCoordinate)) {
+        if (StringUtils.hasLength(startCoordinate) && StringUtils.hasLength(endCoordinate)) {
```

Comment:
```suggestion
        if (!startCoordinate.isBlank() && !endCoordinate.isBlank()) {
```
This is probably better given that it's part of core java.
Alternatively, you could also use `isEmpty()` method.

Reply:
Thank you for your suggestion. hasLength() method does the same job as the deprecated isEmpty(). but isBlank and isEmpty can behave differently. For example when the string is null, the methods from StringUtils can handle it but the methods from String class throw nullPointerException. also in the case of  whitespace strings, isBlank() does the opposite of what StringUtils.isEmpty() does.